unix_integration/common/src/unix_proto.rs

```diff
@@ -177,7 +177,7 @@ pub enum ClientResponse {
     ProviderStatus(Vec<ProviderStatus>),
 
     Ok,
-    Error,
+    Error(String),
```

Comment:
I'm not a fan of String in error types. I think it was a mistake that we did it in the past, so we should probably add stronger types here I think.

Reply:
So swap to an enum?

Reply:
And in that case, should I be using the OperationError enum as the one true error place? 😄

Reply:
(done)